eip712_cosmos.go

```
@@ -1,4 +1,4 @@
-package ante
+package sdk_go
```

golangci-lint (1.62.2)

[warning] var-naming: don't use an underscore in package name

(revive)